components/menu/menu.js

```diff
@@ -67,6 +67,12 @@ class Menu extends HierarchicalViewMixin(LitElement) {
 		this._labelChanged();
 
 		this._onMenuItemsChanged();
+		const slot = this.shadowRoot.querySelector('slot');
```

Comment:
Be sure to check this in Edge - recently a similar query (for a slot element) was failing for dialog footer in Edge, and I think we ended up having to query using a class on the slot instead of the tag name.  Strange... anyway, definitely worth checking if you haven't already.

Reply:
Ah yup - so edge works without the `requestAnimationFrame` in there (but then we need to add back the `role = item.role || item.getAttribute('role')` and so no longer works with the most recent change.

Reply:
Querying by tag name alone should be fine, the dialog case was `.some-class slot` which I'm guessing the shady DOM polyfill wasn't able to handle.